MetaMask/metamask-extension
# Fix for overlapping text in transaction list item

## Explanation
Added fix for overlapping text in transaction list item inside activity tab, when site origin is long there is no more overlapping.

* Fixes #15180

## Screenshots/Screencaps

### Before

![fireFoxBug](https://user-images.githubusercontent.com/120601632/212712401-55c2063f-0d95-4d10-8e58-3ff1cf564171.gif)

### After

![fireFoxBugFix](https://user-images.githubusercontent.com/120601632/212711995-956e1a3a-2688-4fc3-b744-3c641e374496.gif)

## Manual Testing Steps

1. Login to MM on Firefox
2. Deploy a Contract from the [E2E Test Dapp](https://metamask.github.io/test-dapp/)
3. Interact with the contract
4. Open the popup view and inspect the activity log
5. Update the site origin from metamask.github.io to long-name-dapp-metamask.github.io (site origin should be long)
6. Site origin string is now not overlapping, instead it gets shortened with three dots
